junebug/channel.py

```diff
@@ -193,9 +193,17 @@ def _transport_config(self):
 
     @property
     def _application_config(self):
+        redis_manager = self.redis._config
+        # For tests, FAKE_REDIS cannot be an object, else deepcopy doesn't
+        # work.
+        if 'FAKE_REDIS' in redis_manager:
+            redis_manager['FAKE_REDIS'] = 'yes'
```

Comment:
I think that will work. The reason it didn't work before is that we weren't passing the whole config to the channel, so we needed to do things like this. I'll try and see if I can correct this.